supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -401,9 +401,7 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     val jesOutputs: Seq[JesFileOutput] = generateJesOutputs(jobDescriptor) ++ monitoringOutput
 
     instantiateCommand match {
-      case Success(command) =>
-        tellMetadata("instantiatedCommand", command)
-        runWithJes(command, jesInputs, jesOutputs, /* runIdForResumption, */ monitoringScript.isDefined)
+      case Success(command) => runWithJes(command, jesInputs, jesOutputs, /* runIdForResumption, */ monitoringScript.isDefined)
```

Comment:
I did that on the very first version of the JES backend that didn't support recover, and before we knew the nitty gritty of what recover was going to look like.  I think this can go now.  😄